src/python/grapl-common/grapl_common/grapl_logger.py

```diff
@@ -4,8 +4,8 @@
 import sys
 
 
-def get_module_grapl_logger(
-    default_log_level: str = "INFO", log_to_stdout: bool = False
+def get_module_grapl_logger(  # TODO: delete this function
```

Comment:
can we get a proposed solution and/or a stub ticket for this?

i am not attached to this particular solution; I _am_ attached to a mostly-universal way to grab a logger that knows about GRAPL_LOG_LEVEL.

Reply:
The proposed solution is to adhere as closely as possible to the python documentation. In particular:

> Note that Loggers should NEVER be instantiated directly, but always through the module-level function logging.getLogger(name).

https://docs.python.org/3/library/logging.html#logger-objects

So, everywhere we need a logger, we should do the following:
```python
GRAPL_LOG_LEVEL = os.getenv("GRAPL_LOG_LEVEL", "INFO")
LOGGER = logging.getLogger(__name__)
LOGGER.setLevel(GRAPL_LOG_LEVEL)
LOGGER.addHandler(logging.StreamHandler(stream=sys.stdout))
```
there's nothing wrong with a little repetition here. Making this simple and explicit is much more important than making it convenient and clever.